src/client/opamArg.ml

```diff
@@ -1020,6 +1021,15 @@ let config =
     "Without argument, prints a documented list of all available variables. With \
      $(i,PACKAGE), lists all the variables available for these packages. Use \
      $(i,-) to include global configuration variables for this switch.";
+    ["set"]     , `set     , ["VAR";"VALUE"],
```

Comment:
Yes indeed; at the moment it just sets in `global-config.config`, so variable names with colons need to be refused. We don't yet keep package's config files so setting their variables would be more misleading than anything at the moment.